SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialWorldSettings.h

```diff
@@ -22,4 +23,7 @@ class SPATIALGDK_API ASpatialWorldSettings : public AWorldSettings
 	UPROPERTY(EditAnywhere, Config, Category = "Load Balancing")
 	TSubclassOf<UAbstractLockingPolicy> LockingPolicy;
 
+	/** Layer loadbalancing  configuration. */
```

Comment:
NIT space in load balancing, double space before configuration
```suggestion
	/** Layer load balancing configuration. */
```